controllers/nginx/pkg/template/template.go

```diff
@@ -434,12 +437,6 @@ func buildRateLimit(input interface{}) []string {
 		limits = append(limits, limit)
 	}
 
-	if loc.RateLimit.RPM.Limit > 0 {
-		limit := fmt.Sprintf("limit_req zone=%v burst=%v nodelay;",
-			loc.RateLimit.RPM.Name, loc.RateLimit.RPM.Burst)
```

Comment:
Unrelated to this PR, but I have a few questions on this particular section:

1. What happens if`loc.RateLimit.RPM.Limit` > 0, but `loc.RateLimit.RPM.Burst` does not exist?
2. Should we look for `loc.RateLimit.RPM.Burst` instead?